sql/_27_banana_qa/issue_12022_index_cardinality/cases/_005.sql

```diff
@@ -35,23 +35,24 @@ update statistics on t with fullscan;
 show index in t;
 
 --test: index_cardinality
-select index_cardinality('t', 'fidx1', 0);
-select index_cardinality('t', 'fidx1', 2);
-select index_cardinality('t', 'fidx1', 4);
-select index_cardinality('t', 'fidx2', 0);
-select index_cardinality('t', 'fidx2', 3);
-select index_cardinality('t', 'fidx2', 5);
-select index_cardinality('t', 'fidx2', 7);
-select index_cardinality('t', 'fidx2', 9);
-select index_cardinality('t', 'fidx3', 1);
-select index_cardinality('t', 'fidx3', 4);
-select index_cardinality('t', 'fidx3', 6);
-select index_cardinality('t', 'fidx3', 9);
-select index_cardinality('t', 'fidx3', 11);
-select index_cardinality('t', 'fidx3', 13);
-select index_cardinality('t', 'fidx3', 14);
-select index_cardinality('t', 'fidx3', 15);
-select index_cardinality('t', 'fidx3', 16);
+--test: index_cardinality
```

Comment:
Is this comment added by mistake?

Reply:
Reflected the review. Please check again.